services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/BoosterNotificationStructureProvider.java

```diff
@@ -54,16 +61,17 @@ public BoosterNotificationStructureProvider(DistributionServiceConfig distributi
    * Archive will be published in order to not override any previous archive on CDN with broken data. Provided rules are
    * filtered by rule type parameter which could be 'Acceptance', 'Invalidation' or 'BoosterNotification'.
```

Comment:
The last sentence in this javadoc does not apply because it seems booster notification are independently retrieved.